File: matiasmasso/Mat.NET/DTO/DTOBancTransfer.cs
using System;

namespace DTO
{
    public class DTOBancTransfer : DTOBaseGuid
    {
        public DTOBanc BancEmissor { get; set; }
        public DTOCca Cca { get; set; }
        public DTOContact Beneficiari { get; set; }
        public DTOBankBranch BeneficiariBankBranch { get; set; }
        public string Account { get; set; }
        public DTOAmt Amt { get; set; }
        public string Concepte { get; set; }

        public DTOBancTransfer() : base()
        {
        }

        public DTOBancTransfer(Guid oGuid) : base(oGuid)
        {
        }
    }

    public class DTOBancTransferItem : DTOCcb
    {
        public DTOIban Iban { get; set; }
        public string Concept { get; set; }
    }

}

File: matiasmasso/Mat.NET/DT2/DTOBancTransfer.vb
Public Class DTOBancTransfer
    Inherits DTOBaseGuid

    Property BancEmissor As DTOBanc
    Property Cca As DTOCca
    Property Beneficiari As DTOContact
    Property BeneficiariBankBranch As DTOBankBranch
    Property Account As String
    Property Amt As DTOAmt
    Property Concepte As String

    Public Sub New()
        MyBase.New()
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub
End Class
